system-int-test/src/cmr/system_int_test/data2/umm_spec_subscription.clj

```diff
@@ -38,7 +38,11 @@
   ([attribs]
    (subscription attribs latest-umm-sub-verison))
   ([attribs version]
-   (umm-sub/map->UMM-Sub (merge (sample-umm-subscription version) attribs))))
+   (let [sample-subscription (merge (sample-umm-subscription version) attribs)
+         subscription (if (and (contains? attribs :Type) (= (:Type attribs) "collection"))
```

Comment:
This can be simplified, `(if (= "collection" (:Type attribs))`, the `and` shouldn't be needed.